.github/workflows/ipfs-deploy.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
       - name: Build image
         run: |
-          docker build . --file Dockerfile --tag $IMAGE_TAG
+          docker build --target self-host . --file Dockerfile --tag $IMAGE_TAG
```

Comment:
We need a job step (probably the last one) for `docker ebuild --target remote-host`.

Reply:
this should now be reverted in favor of adding parameters to `docker run` strategy